SessionPlayer.js

```diff
@@ -0,0 +1,198 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { View, StyleSheet } from 'react-native';
+
+import PlayerActionButton from 'PlayerActionButton';
+
+/*
+ * Guides
+ */
+import GuideBreathingNormal from 'Guides/Breathing/Normal';
+import GuideBreathingPause from 'Guides/Breathing/Pause';
+import GuideBreathingFinished from 'Guides/Breathing/Finished';
+
+import GuidePowerPoseSuperHuman from 'Guides/PowerPose/SuperHuman';
+import GuidePowerPosePause from 'Guides/PowerPose/Pause';
+import GuidePowerPoseFinished from 'Guides/PowerPose/Finished';
+
+const guides = {
+    Breathing: {
+        Normal: GuideBreathingNormal,
+        Paused: GuideBreathingPause,
```

Comment:
We don't need a unique `Paused` state. All `Paused` state will look the same, so it's just a global component.